spec/models/solidus_afterpay/payment_method_spec.rb

```diff
@@ -140,4 +140,29 @@
       end
     end
   end
+
+  describe "#cache" do
+    let(:memory_store) { ActiveSupport::Cache.lookup_store(:memory_store) }
+    let(:cache) { Rails.cache }
+
+    before do
+      allow(Rails).to receive(:cache).and_return(memory_store)
+      Rails.cache.clear
+    end
+
+    it "retrieves the minimum amount from cache" do
+      cache.write("solidus_afterpay_min_amount", 1.0)
+      expect(described_class.new.send(:minimum_amount)).to eq(1.0)
+    end
+
+    it "retrieves the maximum amount from cache" do
+      cache.write("solidus_afterpay_max_amount", 1000.0)
+      expect(described_class.new.send(:maximum_amount)).to eq(1000.0)
```

Comment:
it's now good practice to test `private` methods. You should test the `available_for_order?` method with different caching scenarios instead (with and without cache hit).

Reply:
Done!